### PR TITLE
Add price.adddress again for pools

### PR DESCRIPTION
**Pull request by kenbodnar**
## Description

Aquarius issue #388 requested removing price.address and replacing it with price.exchange_id. However price.address is needed for pools. It was added back in for the appropriate cases.

## Is this PR related with an open issue?

Related to Issue #

## Types of changes

- [x ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
 

#### Funny gif

Put a link of a funny gif inside the parenthesis-->

### CODE REVIEW

File: aquarius/events/metadata_updater.py [hunk 1]
```diff
@@ -461,13 +461,14 @@ def _get_price_updates_from_fixed_rate_exchange(
...
                 f"address={self.ex_contract.address} and type as empty string."
             )
-            price_dict.update({"address": self.ex_contract.address, "type": "exchange"})
+            price_dict.update({"exchange_id": exchange_id, "type": "exchange"})
+            price_dict.update({"address": "", "type": ""})
         else:
             logger.info(
                 "Found price=None, setting address and type as empty string."
             )  # noqa
             price_dict.update({"address": "", "type": ""})
```
Comment (calina-c) [inline aquarius/events/metadata_updater.py:470]: these two lines can be combined. The "type" key and its empty value are just repeated. Please add "exchange_id": "" to the original update instruction.

**Review by calina-c — CHANGES_REQUESTED** [review]

**Comment by calina-c** [comment]
@reviewers: I just combined Ken's dict update statements into one, but I also did a tiny change. Not sure if it is correct. In one instance, he was setting the "type" to "" but exchange_id was populated. I assumed type needs to be "exchange" if an "exchange_id" is set. Is that correct? 

https://github.com/oceanprotocol/aquarius/pull/424/commits/67a56188366598dfe03a982c379cd8ff007b30ad

**Comment by alexcos20** [comment]
Let's keep it as it is for now, from our market you can create either a pool or a fre.
Will continue later

**Review by calina-c — APPROVED** [review]

**Comment by DMats** [comment]
The changes in this PR fix #410

**Comment by DMats** [comment]
Closes #410

(trying to get Zenhub to link them together)

**Pull request merged**